src/workerd/server/server.c++

```diff
@@ -1168,7 +1168,9 @@ public:
       auto content = kj::str("{\"Browser\": \"workerd\", \"Protocol-Version\": \"1.3\" }");
       auto out = response.send(200, "OK", responseHeaders, content.size());
       co_return co_await out->write(content.begin(), content.size());
-    } else if (url.endsWith("/json") || url.endsWith("/json/list")) {
+    } else if (url.endsWith("/json") ||
+               url.endsWith("/json/list") ||
+               url.endsWith("/json/list?for_tab")) {
```

Comment:
consider using the URL parser here instead to avoid any issues in the future

Reply:
If you have a clear view of what that would look like, feel free to update the code. I've taken a look and I am not sure we can predict the intent of future changes in the inbound URL, nor that the kj::URL class buys much here (url is the string from a `HTTP_REQUEST`).

We don't currently have any tests here at all. The inbound request leading to this change comes from whatever Chrome someone has installed. :-(